src/views/AnalysisModule/index.tsx

```diff
@@ -111,6 +170,29 @@ function AnalysisModule(props: AnalysisModuleProps) {
         },
     );
 
+    const [
+        ,
+        overviewResponse,
+        ,
+        ,
+    ] = useRequest<MultiResponse<AnalysisOverview>>(
+        {
+            url: `server://projects/${activeProject}/analysis-overview/`,
+            method: 'GET',
+            autoTrigger: true,
+            onFailure: (_, errorBody) =>
+                notifyOnFailure(_ts('analysis', 'analysisModule'))({ error: errorBody }),
+        },
+    );
+
+    const piechartData: AuthoringOrganizations = overviewResponse?.authoringOrganizations ?? [];
```

Comment:
shouldn't it be as overviewResponse?.results.authoringOrganizations ?. If not then you should remove <MultiResponse<T>> type from the corresponding useRequest. The type for pieChartData should be AuthoringOrganizations[].